### PR TITLE
Task 2. Components.

### PR DESCRIPTION
**Pull request by MaryBozhyk**

### CODE REVIEW

File: src/app/courses-page/add-course-button/add-course-button.component.ts [hunk 1]
```diff
@@ -1,15 +1,20 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
 
 @Component({
   selector: 'app-add-course-button',
   templateUrl: './add-course-button.component.html',
   styleUrls: ['./add-course-button.component.scss']
 })
 export class AddCourseButtonComponent implements OnInit {
```
Comment (SteveRusin) [inline src/app/courses-page/add-course-button/add-course-button.component.ts]: seems like it's a simple button. I don't think it worth creating a separate component for it. If you want to put some logic into the button better to use a directive. Or you can take an angular material approach where they create a [button component](https://github.com/angular/components/blob/master/src/material/button/button.ts#L67) with attribute selector and simply project the content inside with ripple effect. Using attribute selector you can preserve browser accessibility to the button

File: src/app/courses-page/add-course-button/add-course-button.component.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@import "variables";
+
+:host {
+    display: block;
+    margin: 0 85px 15px auto;
+}
+
+.add-btn {
+    border-radius: 3px;
+	background: color(pacificBlue);
```
Comment (SteveRusin) [inline src/app/courses-page/add-course-button/add-course-button.component.scss]: nit: could you alight the indentation? I would suggest using 2 space indentation and some kind of prettier to format the code.

File: src/app/courses-page/add-course-button/add-course-button.component.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+@import "variables";
+
+:host {
+    display: block;
+    margin: 0 85px 15px auto;
```
Comment (SteveRusin) [inline src/app/courses-page/add-course-button/add-course-button.component.scss]: it's a bad approach to adding margins to the component itself, you will end up overriding this margin over and over again in different places. The component which uses it should dictate the margins.

File: src/app/courses-page/add-course-button/add-course-button.component.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    font-weight: 600;
+    border: none;
+    
+    &:hover {
+        cursor: pointer;
+        background-color: color(deepSkyBlue1);
+    }
+
+    &:focus {
+        outline: none;
```
Comment (SteveRusin) [inline src/app/courses-page/add-course-button/add-course-button.component.scss]: by removing the outline you're removing accessibility from the button
Here is the good video about that
https://www.youtube.com/watch?v=Mvu5OMGcdVA&t=5s&ab_channel=KevinPowell

File: src/app/courses-page/load-more/load-more.component.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-load-more',
+  templateUrl: './load-more.component.html',
+  styleUrls: ['./load-more.component.scss']
+})
+export class LoadMoreComponent implements OnInit {
```
Comment (SteveRusin) [inline src/app/courses-page/load-more/load-more.component.ts]: again, it's a simple button. Not worth creating a separate component

File: src/app/shared/components/login-button/login-button.component.html [hunk 6]
```diff
@@ -1,3 +1,3 @@
 <button class="button">
-    <a href="#"><img src="assets/icons/ic_person.svg" alt="img" class="icon"> Login</a>
+   <img src="assets/icons/ic_person.svg" alt="img" class="icon"> Login
```
Comment (SteveRusin) [inline src/app/shared/components/login-button/login-button.component.html]: no need for a separate component for a simple button

File: src/app/courses-page/add-course-button/add-course-button.component.ts (hunk 1 above)
Comment (MaryBozhyk) [reply src/app/courses-page/add-course-button/add-course-button.component.ts]: @SteveRusin button was moved to courses-page template

File: src/app/courses-page/add-course-button/add-course-button.component.scss (hunk 2 above)
Comment (MaryBozhyk) [reply src/app/courses-page/add-course-button/add-course-button.component.scss]: @SteveRusin prettier is used

File: src/app/courses-page/add-course-button/add-course-button.component.scss (hunk 3 above)
Comment (MaryBozhyk) [reply src/app/courses-page/add-course-button/add-course-button.component.scss]: @SteveRusin these margins are removed

File: src/app/courses-page/add-course-button/add-course-button.component.scss (hunk 4 above)
Comment (MaryBozhyk) [reply src/app/courses-page/add-course-button/add-course-button.component.scss]: @SteveRusin Thanks. Default outline was changed to box-shadow and appropriate mixin was created.

File: src/app/courses-page/load-more/load-more.component.ts (hunk 5 above)
Comment (MaryBozhyk) [reply src/app/courses-page/load-more/load-more.component.ts]: @SteveRusin button was moved to courses-list template

File: src/app/shared/components/login-button/login-button.component.html (hunk 6 above)
Comment (MaryBozhyk) [reply src/app/shared/components/login-button/login-button.component.html]: @SteveRusin login and logout buttons were moved to LoginButtonsComponent

**Pull request merged**